### PR TITLE
docs: Updating Arc42 document

### LINKED ISSUE #94: TRG 7.07: Remove file header from md file and add NOTICE as post or preamble instead
### Description

Remove file header and and add NOTICE as post or preamble instead https://github.com/eclipse-tractusx/sd-factory/blob/main/docs/ARC42.md
[TRG 7.07](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-07)

### PR DESCRIPTION
**Pull request by adkumar1**
- Fixes #94 
- Removed the Header from ARC42 document

### CODE REVIEW

File: docs/ARC42.md
Comment (SebastianBezold) [inline docs/ARC42.md:1]: Hi @adkumar1, please read [TRG 7.07](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-07#documentation) carefully. 
You do need to state licence info in your documentation. But as Notice section, that can be read by someone reading your docs

**Review by SebastianBezold — CHANGES_REQUESTED** [review]
Hi @adkumar1,

besides my comment on the Notice section for our docs. Please clean up this PR. You state, that it is fixing a single issue (which it should), but you include a whole bunch of other stuff. Plaese create isolated PRs with fixes instead of bundling everything together

File: docs/ARC42.md
Comment (adkumar1) [reply docs/ARC42.md:1]: @sharathshivprasad : Please have a look

**Comment by adkumar1** [comment]
> Hi @adkumar1,
> 
> besides my comment on the Notice section for our docs. Please clean up this PR. You state, that it is fixing a single issue (which it should), but you include a whole bunch of other stuff. Plaese create isolated PRs with fixes instead of bundling everything together

Will do that .

**Comment by adkumar1** [comment]
Closing this PR as updated separately in PR #102

**Pull request closed without merging**